boards/arm/raytac_mdbt50q_db_33_nrf52833/doc/index.rst

```diff
@@ -0,0 +1,207 @@
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features have not been enabled yet for this board.
+See `MDBT50Q-DB-33 website`_ and `MDBT50Q-DB-33 Specification`_
+for a complete list of Raytac MDBT50Q-DB-33 board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.13
+* LED2 (red) = P0.14
+* LED3 (blue) = P0.15
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P0.11
+* BUTTON2 = SW2 = P0.12
+* BUTTON3 = SW3 = P0.24
+* BUTTON4 = SW4 = P0.25
+
+UART
+----
+* RXD = P0.08
+* TXD = P0.06
+* RTS = P0.05
+* CTS = P0.07
+
+Programming and Debugging
+*************************
+
+Applications for the ``raytac_mdbt50q_db_33_nrf52833`` board configuration can be
+built, flashed, and debugged in the usual way. See
+:ref:`build_an_application` and :ref:`application_run` for more details on
+building and running.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the board nRF52840 DK
```

Comment:
This will not work, I checked the schematic and was assuming that there was a UART <-> USB bridge but there isn't, so there will be no usable UART. You'd likely have to do similar to this if you wanted to use USB CDC. If not, then these instructions need updating with how someone would use the UART.

- USB CDC device: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts#L301
- USB CDC default as chosen: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts#L10
- Backend: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/thingy53_nrf5340/Kconfig#L32
- CDC configuration (note: do not set product, VID or PID): https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/thingy53_nrf5340/Kconfig.defconfig#L85

Reply:
I just want to let users know how to burn the program into the demo board. I'll add instructions for using a USB to TTL converter.